packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -4,6 +4,8 @@
 
 import 'dart:async';
 
+import 'package:multicast_dns/multicast_dns.dart';
```

Comment:
I believe this will need to be added to the BUILD.gn as well

Reply:
Added - can you verify I did it right though?

Reply:
cc @zanderso do we need to manually roll this into fuchsia?

Reply:
Yes, after the in-flight roll lands, this will cause the auto-roller to block until Fuchsia's third_party Dart packages are manually updated. I can help with that.

Also, it should go in this one https://github.com/flutter/flutter/blob/master/packages/flutter_tools/BUILD.gn, not the one for flutter_test.

Reply:
Thanks for catching that.  I've moved it to the correct BUILD.gn

Reply:
Is this good to land now, or were we waiting on confirmation for any of this?

Reply:
This is okay to land from the Fuchsia perspective. An earlier flutter/flutter change also requires an update to the Fuchsia third_party Dart packages, so it would be convenient to handle this one and that one at the same time.